modules/ppcp-save-payment-methods/resources/js/add-payment-method.js

```diff
@@ -95,11 +95,11 @@ import {
 						) {
 							return;
 						}
-
+                        document.getElementById('place_order').disabled = true;
```

Comment:
Better check is not null before accessing property:
```
 document.getElementById('place_order')?.disabled = true;
```

Reply:
The element must exist now because we also use it for `addEventListener`. and call it inside of it.